build.sbt

```diff
@@ -255,6 +255,7 @@ lazy val scalahostSbt = project
     buildInfoSettings,
     sbt.ScriptedPlugin.scriptedSettings,
     sbtPlugin := true,
+    publishMavenStyle := false, // necessary for pre-releases to work with addSbtPlugin
```

Comment:
I never got to reinstating this part of our build. Currently doing this manually (ugh!).